packages/@webex/plugin-meetings/test/unit/spec/reachability/request.js

```diff
@@ -114,5 +129,37 @@ describe('plugin-meetings/reachability', () => {
       assert.deepEqual(res.joinCookie, {anycastEntryPoint: "aws-eu-west-1"})
       assert.notCalled(webex.internal.newMetrics.callDiagnosticLatencies.measureLatency);
     });
+
+    it('sends a POST request with the correct params when appVersion is undefined', async () => {
+      // Mocking appType & appVersion to undefined
+      webex.config.support.appType = undefined;
+      webex.config.support.appVersion = undefined;
+      appType = webex?.config?.support?.appType;
```

Comment:
what is the purpose of this line and next? both `appType` and `appVersion` are not used in this test